### DISCUSSION TITLE
For anyone interested in using Stencil.js with Nano Stores, I created an integration library

### DISCUSSION THREAD

**Discussion by jake-danton** [ai/nanostores#245] [Show and tell]
Check out the [NPM Package](https://www.npmjs.com/package/@giant-leaps/nanostores-stencil) or [GitHub repo](https://github.com/giant-leaps/nanostores-stencil) and feel free to give us feedback or contribute.

**Comment by ai** [comment]
Wow! Looks great. I am going to create a few issues.
